spec/requests/api/v1/payments_spec.rb

```diff
@@ -0,0 +1,315 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+
+RSpec.describe 'payments API', type: :request do
+  let!(:user_params) do
```

Comment:
I did a tiny refactor of the associated model records in #175 if you want to take a look and make this less verbose :)